class django_helper_exceptions:
    pass